src/test/unit_tests/test_nsfs_access.js

```diff
@@ -66,29 +75,22 @@ mocha.describe('new tests check', async function() {
         }
     });
     mocha.it('NON ROOT 1 mkdir - failure', async function() {
+        const new_folder = "folder1";
+        const new_path = path.join(full_path_root, new_folder);
         try {
-            const mkdir_res = await nb_native().fs.mkdir(NON_ROOT1_FS_CONFIG, full_path_non_root, 0o770);
+            const mkdir_res = await nb_native().fs.mkdir(NON_ROOT1_FS_CONFIG, new_path, 0o770);
             assert.fail(`non root has access to mkdir under root dir ${mkdir_res}`);
         } catch (err) {
             assert.equal(err.code, 'EACCES');
         }
     });
     mocha.it('ROOT readdir - dir created by non dir - success', async function() {
-        try {
-            const root_entries = await nb_native().fs.readdir(ROOT_FS_CONFIG, full_path_non_root);
-            assert.fail(`root has access to a folder that should not exist - ${root_entries}`);
-        } catch (err) {
-            assert.equal(err.code, 'ENOENT');
-        }
+            await nb_native().fs.readdir(ROOT_FS_CONFIG, full_path_non_root);
     });
 
     mocha.it('NON ROOT 1 readdir - success', async function() {
-        try {
-            const non_root_entries = await nb_native().fs.readdir(NON_ROOT1_FS_CONFIG, full_path_non_root);
-            assert.fail(`non root 1 has access to a folder created by root with 770 perm - ${p} - ${non_root_entries}`);
-        } catch (err) {
-            assert.equal(err.code, 'EACCES');
-        }
+        const non_root_entries = await nb_native().fs.readdir(NON_ROOT1_FS_CONFIG, full_path_non_root);
```

Comment:
please don't change existing tests that were supposed to fail, add new tests for the supplemental groups tests

Reply:
sure, but by test name it was suppose to succeed. I assume that the test purpose was to test that non root 1 should be able to access folders it was allowed to